client/js/components/week/week-header.js

```diff
@@ -1,13 +1,39 @@
 /** @jsx React.DOM */
 var React = require('react');
+var ViewActionCreators = require('../../actions/view-action-creator');
+var moment = require('moment');
+var _ = require('underscore');
 
 var WeekHeader = React.createClass({
+  getInitialState: function() {
```

Comment:
is there any reason for the week header to have it's own state? this could be 'getDefaultProps' and have the date be in through props from week.js